core/src/main/scala/org/apache/spark/ui/SparkUI.scala

```diff
@@ -135,7 +133,7 @@ private[spark] class SparkUI private (
 private[spark] abstract class SparkUITab(parent: SparkUI, prefix: String)
   extends WebUITab(parent, prefix) {
 
-  def appName: String = parent.getAppName
+  def appName: String = parent.appName
```

Comment:
This can just be removed then, because this becomes a no-op. And then getAppName in the superclass can be removed. It's a pointless getter.